networks/mainnet/network-performance.md

```diff
@@ -38,4 +38,4 @@ In both methods, the data retrieval process after a retrieval deal is accepted i
 2. **Unsealing (if needed)**: The storage provider unseals (decodes) the data so that the requester can read it. Sealing and unsealing are symmetric processes, which means they take roughly the same amount of time in either direction. Thus, the unsealing step is estimated to take around as long as the sealing step listed above, or around \~3 hours for a 32 GiB sector on a machine running minimum hardware requirements.
 3. **Data transfer**: The storage provider begins transferring the data back to the data requester. This is also transferred back at a rate similar to the original data transfer rate, which depends on several factors.
 
-Because of the various steps involved in the data retrieval process, Filecoin storage currently meets similar performance bars as traditional _warm_ or _cold_ storage. To get performance that is similar to other hot storage solutions, most users utilize Filecoin with a caching layer such as IPFS. These hybrid and multi-tiered storage solutions use IPFS for hot storage and Filecoin for affordable, frequent, and versioned backups. Some example hybrid storage products include [Powergate](https://blog.textile.io/filecoin-developer-tools-concepts/) and [Textile Buckets](https://docs.textile.io/buckets/).
+Because of the various steps involved in the data retrieval process, Filecoin storage currently meets similar performance bars as traditional _warm_ or _cold_ storage. To get performance that is similar to other hot storage solutions, most users utilize Filecoin with a caching layer such as IPFS. These hybrid and multi-tiered storage solutions use IPFS for hot storage and Filecoin for affordable, frequent, and versioned backups. Some example hybrid storage products include [Powergate](https://docs.textile.io/powergate/) and [Textile Buckets](https://docs.textile.io/buckets/).
```

Comment:
@robertagora - Textile's Powergate and Textile Buckets (and Estuary) are now deprecated technologies IIUC. Right now, it's probably best to point developers to Lighthouse.storage or web3.storage I think for hot storage layers, best to check with @trruckerfling.

Reply:
Noted, will resolve.